dir.props

```diff
@@ -44,13 +44,18 @@
     <LocalBlobStorageRoot>$(IntermediatePath)blobs/</LocalBlobStorageRoot>
     <LocalBuildInfoRoot>$(IntermediatePath)build-info/</LocalBuildInfoRoot>
     <LocalNuGetPackagesRoot>$(IntermediatePath)nuget-packages/</LocalNuGetPackagesRoot>
+    <SourceBuiltBlobFeedDir>$(IntermediatePath)blob-feed/</SourceBuiltBlobFeedDir>
+    <SourceBuiltPackagesPath>$(SourceBuiltBlobFeedDir)packages/</SourceBuiltPackagesPath>
```

Comment:
Why path here and not dir?

Reply:
I didn't rename some existing args to keep the delta smaller--would make sense to me to do renames for convention in a separate PR, depending on how much people mind merge conflicts.

Reply:
Agreed - I think we can do this on a separate PR to minimize churn for now.